tls-attestation/draft-fossati-tls-attestation.md

```diff
@@ -94,32 +94,61 @@ informative:
 
 --- abstract
 
-Various attestation technologies have been developed and formats have been standardized.
-Examples include the Entity Attestation Token (EAT) and Trusted Platform Modules (TPMs).
-Once attestation information has been produced on a device it needs to be communicated
-to a relying party. This information exchange may happen at different layers in the 
-protocol stack.
+Attestation is the process by which an entity produces evidence about itself
+that another party can use to evaluate the trustworthiness of that entity.
 
-This specification provides a generic way of passing attestation information in the 
-TLS handshake.
+In use cases that require the use of remote attestation, such as confidential computing,
+an attester has to convey evidence or attestation results to a relying party. This 
+information exchange may happen at different layers in the protocol stack. 
 
---- middle
+This specification provides a generic way of passing evidence and attestation results
+in the TLS handshake. Functionality-wise this is accomplished with the help of key
+attestation.
 
+--- middle
 
 #  Introduction
 
 Attestation is the process by which an entity produces evidence about itself
-that another party can use to evaluate the trustworthiness of that entity. 
-One format of encoding evidence is standardized with the Entity Attestation 
-Token (EAT) {{I-D.ietf-rats-eat}} but there are other formats, such as 
-attestation produced by Trusted Platform Modules (TPMs) {{TPM1.2}} {{TPM2.0}}.
-
-This specification defines how to convey attestation information in the 
-TLS handshake with different encodings being supported. This specification
-standardizes two attestation formats -- EAT and TPM-based attestation.
-
-Note: This initial version of the specification focuses on EAT-based attestation.
-Future versions will also define TPM-based attestation.
+that another party can use to evaluate the trustworthiness of that entity.
```

Comment:
Do you still need line 112/113, given the fact that you have already used them at line 100.